server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/java/org/apache/james/jmap/memory/MemoryGetMessageListMethodTest.java

```diff
@@ -49,4 +49,10 @@ protected void await() {
     @Test
     public void getMessageListShouldIncludeMessagesWhenTextFilterMatchesBodyWithStemming() {
     }
+
+    @Override
+    @Ignore("MemoryJmapTestRule doesn't use ElasticSearch Indexing")
+    @Test
+    public void searchByFromFieldDoesntSupportUTF8FromName() {
```

Comment:
Memory should support UTF-8 in from - and if it is not the case we have another bug

Reply:
Seems like the MessageSearches used in SimpleMessageSearchIndex doesn't search on From name, but only on From Address.

Reply:
Then you should update the message accordingly.

Reply:
Can we:
 - 1. Write a test in AbstractMessageSearchIndexTest about searches on FROM names that is passing in ElasticSearch and failing on Memory?
 - 2. At the very least write issues about this behaviour - prefix the @Disabled message with the JIRA ticket and do not forget the GITLAB bug repport as well with the *bug* label.